pkg/api/api.go

```diff
@@ -194,3 +207,51 @@ func (api *api) WatchDelete() {
 		}
 	}
 }
+
+type Version struct {
+	Blackbeard string `json:"blackbeard"`
+	Kubernetes string `json:"kubernetes"`
+	Kubectl    string `json:"kubectl"`
+}
+
+func (api *api) GetVersion() (*Version, error) {
+	cmd := exec.Command("/bin/sh", "-c", "kubectl version --output json")
+	cmdReader, _ := cmd.StdoutPipe()
+
+	err := cmd.Start()
+
+	if err != nil {
+		return nil, err
+	}
+
+	data, err := ioutil.ReadAll(cmdReader)
+
+	if err != nil {
+		return nil, err
+	}
+
+	type response struct {
+		ClientVersion struct {
+			Major string `json:"major"`
+			Minor string `json:"minor"`
+		} `json:"clientVersion"`
+		ServerVersion struct {
+			Major string `json:"major"`
+			Minor string `json:"minor"`
+		} `json:"serverVersion"`
+	}
+
+	var v response
+
+	err = json.Unmarshal(data, &v)
+
```

Comment:
This part should be in the `kubernetes` package as it use kubectl which a "infrastructure" choice. One day, we may  call the Kubernetes REST API to get the version, for example.

(I'm talking about the whole part that get the kubectl & kuberntes version)